### DISCUSSION TITLE
Hammerdb to generate and ingest time series data to Pubsub

### DISCUSSION THREAD

**Discussion by noopur100392** [TPC-Council/HammerDB#465] [General]
Hi All, 

I need to ingest time series data to bigquery via pubsub.
I need to understand if I can somehow use hammerdb to generate time-series data and send to Pubsub topic..

the only requirement for me is to generate time-series data and send to pusbub, as core-iot is on demise path hence I was thinking If I can use hammerdb instead to generate it.

If any one can help me with this...?? thank you in advance :)

**Comment by sm-shaw** [comment]
Your best option with hammerdb is to use the datagen feature to generate flat files of TPROC-C or TRPOC-H data and then use this for the testing with other tools.
